core/os/memory.cpp

```diff
@@ -80,6 +80,7 @@ void *Memory::alloc_static(size_t p_bytes, bool p_pad_align) {
 #endif
 		return s8 + PAD_ALIGN;
 	} else {
+		// cppcheck-suppress memleak
```

Comment:
This is a false positive. This is the memory allocation function the engine uses. The memory pointer is not freed because it is transferring ownership of the pointer to the caller. I can explain that in a comment block.